node/node.go

```diff
@@ -491,3 +587,152 @@ func (nn nexNode) EmitEvent(inNamespace string, inEvent json.RawMessage) error {
 	}
 	return nn.nc.Publish(models.EventAPIPrefix+"."+inNamespace, event)
 }
+
+func (nn nexNode) StoreRunRequest(workloadType, inId string, inRequest *actorproto.StartWorkload) error {
+	pub, err := nn.iNatsNkey.PublicKey()
+	if err != nil {
+		return err
+	}
+
+	nc, err := nats.Connect(nn.iNatsURL, nats.Nkey(pub, func(nonce []byte) ([]byte, error) {
```

Comment:
Any reason to not have a long-lived connection set on `nexNode` at startup time that can be reused for the methods that need it?

Reply:
I would have to go through and see how the internal nats server works, its one of the pieces that didn't get a full rewrite and im not sure if its creating conns on demand